README.md

```diff
@@ -1 +1,60 @@
-# Hello-microverse-review
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+
```

Comment:
- [ ] It would be nice if you could include a one-paragraph statement about the project in your README file to make it look more professional.